db/migrations/state/0010.sql

```diff
@@ -0,0 +1,19 @@
+-- +migrate Up
```

Comment:
Can you add the test for this migration?

Reply:
yes, done!